conda/activate.py

```diff
@@ -81,11 +83,11 @@ def get_export_unset_vars(self, export_metavars=True, **kwargs):
         """
         :param export_metavars: whether to export `conda_exe_vars` meta variables.
         :param kwargs: environment variables to export.
-                       .. if you pass and set any other variable to None, then it
-                       emits it to the dict with a value of None.
+                        .. if you pass and set any other variable to None, then it
+                        emits it to the dict with a value of None.
 
         :return: A dict of env vars to export ordered the same way as kwargs.
-                 And a list of env vars to unset.
+                And a list of env vars to unset.
```

Comment:
Why are we making these changes?

Reply:
This is what I was referencing in the second bullet point. The indentation was one space off.

Reply:
lets actually follow the standard hanging indent policy instead (see https://stackoverflow.com/a/50455731/5239932)

Reply:
The issue was because the spacing should be in multiples of four. I took care not to change the indent level. There were three spaces instead of four in two cases and five spaces instead of four in one case.

Reply:
gotcha, lets abide by standard hanging indent policy instead, so only indent the hanging line once:

```
        :param kwargs: environment variables to export.
            .. if you pass and set any other variable to None, then it
            emits it to the dict with a value of None.
        :return: A dict of env vars to export ordered the same way as kwargs.
            And a list of env vars to unset.
```

Reply:
OK. I'll update the pull request and the code to reflect that I'm also making that fix.